AABB_tree/include/CGAL/AABB_face_graph_triangulated_primitive.h

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2012 INRIA Sophia-Antipolis (France).
```

Comment:
As is, this file cannot go into master. The documentation of the package AABB_tree is inside the source so this new class will come up into the documentation. The doc seems to be only a copy-paste of `AABB_face_graph_triangle_primitive`.
There is no need here to inherit from `AABB_primitive` (thus the changes in the class should be undo). Simply have a look at the [`AABBPrimitive`](http://doc.cgal.org/latest/AABB_tree/classAABBPrimitive.html) concept and provide needed members. No property map is needed. I suggest to put this file in the include directory of the polyhedron demo. Feel free to contact me if you have any question.